src/showcase/index.md

```diff
@@ -42,29 +46,51 @@ toc: false
         {% for case in site.data.showcases %}
             {% capture modulo3 %}{{ forloop.index0 | modulo:3 }}{% endcapture %}
             {% if modulo3 == '0' %}
-                <div class="showcase__apps__cards text-left card-deck">
+                <div class="showcase__apps__cards text-left card-deck flex-column flex-lg-row">
             {% endif %}
                     <div class="card">
+                        <div class="card-header">
+                            {% if case.logo_src %}
+                                {% asset '{{ case.logo_src }}' class='showcase__apps__logo' %}
+                            {% endif %}
```

Comment:
It looks strange to have the logo above the name, esp. when the logo is a lockup containing the same text as the name. In those cases, it might be more visually appealing to drop the text name and show a larger lockup.